config/deb.am

```diff
@@ -38,9 +43,14 @@ if CONFIG_USER
 	pkg7=$${name}-test-$${version}.$${arch}.rpm; \
 	pkg8=$${name}-dracut-$${version}.$${arch}.rpm; \
 	pkg9=$${name}-initramfs-$${version}.$${arch}.rpm; \
+	path_prepend=`mktemp -d /tmp/intercept.XXX`; \
+	ln -s /bin/true $${path_prepend}/dh_shlibdeps; \
```

Comment:
I've dismissed my review because i cannot approve this change. If this is a real concern we should put it as a workaround in the wiki, shipping this new hacky fix doesn't sound good to me.

> There may be some way to pass excluded files through the alien call, into its debian directory, but that honestly buys us so little, and would cost so much time to investigate.

Honestly, not so much:

```patch
diff --git a/config/deb.am b/config/deb.am
index 648417b2a..de3972366 100644
--- a/config/deb.am
+++ b/config/deb.am
@@ -38,6 +38,11 @@ if CONFIG_USER
        pkg7=$${name}-test-$${version}.$${arch}.rpm; \
        pkg8=$${name}-dracut-$${version}.$${arch}.rpm; \
        pkg9=$${name}-initramfs-$${version}.$${arch}.rpm; \
+       path_prepend=`mktemp -d /tmp/intercept.XXX`; \
+       echo "`which dh_shlibdeps` -- -xlibuutil1linux -xlibnvpair1linux -xlibzfs2linux -xlibzpool2linux" > $${path_prepend}/dh_shlibdeps; \
+       chmod +x $${path_prepend}/dh_shlibdeps; \
+       env PATH=$${path_prepend}:$${PATH} \
        fakeroot $(ALIEN) --bump=0 --scripts --to-deb \
            $$pkg1 $$pkg2 $$pkg3 $$pkg4 $$pkg5 $$pkg6 $$pkg7 \
            $$pkg8 $$pkg9;
```